Adyen/httpclient.py

```diff
@@ -175,6 +176,8 @@ def _requests_post(self, url,
         auth = None
         if username and password:
```

Comment:
Also here the api key can be the default and not the fallback plan

Reply:
I am not sure what you mean default. Do you suggest to remove basic authentication?

Reply:
I agree with Attilla ApiKey has preference over username and password.

Reply:
It is fixed by adding the xapikey in high priority